helpers/loadOriginalLockType.js

```diff
@@ -16,7 +16,7 @@ async function loadOriginalLockType(CreatedLock) {
     }
 
     const Reds = await RandomInt(LockDetails.Variable_Min_Reds, LockDetails.Variable_Max_Reds);
```

Comment:
I considered refactoring this to use the card-game code, but couldn't see an easy way to do it.  An issue that arose at several points, including this one, is that the database fields are not arrays.  In order to use the card generation code in card-game we would need to retrieve the fields from the LockOriginalType db, create the LockConfig object with the min and max CardMapping objects and set the fields, then create a Lock object with that LockConfig object, and call the createCards method on the Lock object, and then put the results into an LoadedOriginalLock object for storage in the database.  It could definitely be done, but I figured it was best to leave the pre-existing code alone except for fixing bugs.